Catalog/Examples/Combine/Progressable/CombineProgressableViewController.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  CombineProgressableViewController.swift
+//  Catalog
+//
+//  Created by Zvonimir Medak on 23.11.2021..
+//  Copyright © 2021 Infinum. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import Combine
+import CombineExt
+
+@available(iOS 13, *)
+final class CombineProgressableViewController: UIViewController, Progressable {
+
+    private var subscriptions: Set<AnyCancellable> = []
+    private let inputSubject = ReplaySubject<String, ProgressableError>(bufferSize: 3)
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        handle(inputSubject)
+        handleProgressable()
+    }
+
+}
+
+@available(iOS 13, *)
+private extension CombineProgressableViewController {
+
+    func handle(_ inputSubject: ReplaySubject<String, ProgressableError>) {
```

Comment:
You don't need to pass the input subject since it's a property of the VC 🙂 

```suggestion
    func handle() {
```

After doing this, consider renaming handle to something more descriptive as well 😄